### PR TITLE
Naming fixes

### PR DESCRIPTION
**Pull request by LAGIdiot**
[Added] missing language entries for Basalt, Mineral, Concrete, Machine Casing, Metal Casing, Boiler Casing, Hazard Signs, Wire Coils, some tools & recipe maps
[Fixed] language entries for Light Blue Dye, some Conveyors, Mc Guffium, some tools

### CODE REVIEW

File: src/main/java/gregtech/common/items/MetaTool.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public void registerSubItems() {
         CROWBAR = addItem(10, "tool.crowbar").setToolStats(new ToolCrowbar()).addOreDict(ToolDictNames.craftingToolCrowbar).addToList(GregTechAPI.crowbarList);
         SCREWDRIVER = addItem(11, "tool.screwdriver").setToolStats(new ToolScrewdriver()).addOreDict(ToolDictNames.craftingToolScrewdriver).addToList(GregTechAPI.screwdriverList);
         MORTAR = addItem(12, "tool.mortar").setToolStats(new ToolMortar()).addOreDict(ToolDictNames.craftingToolMortar);
-        WIRECUTTER = addItem(13, "tool.wirecutter").setToolStats(new ToolWireCutter()).addOreDict(ToolDictNames.craftingToolWireCutter);
+        WIRECUTTER = addItem(13, "tool.wire_cutter").setToolStats(new ToolWireCutter()).addOreDict(ToolDictNames.craftingToolWireCutter);
```
Comment (Exidex) [inline src/main/java/gregtech/common/items/MetaTool.java]: Does wire cutter still has texture?

Comment (LAGIdiot) [reply src/main/java/gregtech/common/items/MetaTool.java]: No it doesn't. Fixed in next commit.

Comment (LAGIdiot) [reply src/main/java/gregtech/common/items/MetaTool.java]: My bad

**Comment by Exidex** [comment]
While you are at it you might as well add i18n for other things like machines

**Comment by LAGIdiot** [comment]
Some mechines has broken localization because of name.name on the end and I did not figure out this yet. I will look at this later.

Well I can do that, it will probably not be done before weekend but sure I will do it.

Let's leave this pull request open so you can track my progress. And everyone knows I am working on it.

**Comment by LAGIdiot** [comment]
It seems automatic sync fu*k it up somehow together...

**Comment by Exidex** [comment]
Looks good to me

**Pull request merged**